weave/wandb_util.py

```diff
@@ -105,8 +105,13 @@ def _convert_type(old_type: Weave0TypeJson) -> types.Type:
         return types.Int()
     elif old_type_name == "ndarray":
         # return ops.LegacyTableNDArrayType()
-        # Just return NoneType. The data is None.
-        return types.NoneType()
+        if old_type.get("params", {}).get("serialization_path") is not None:
+            # Just return NoneType. The data is None.
+            return types.NoneType()
+
+        # SDK converted the array to a pylist
+        return types.List(types.UnknownType())
```

Comment:
Why Unknown? Isn't it List(list(list... of number?

Reply:
We dont have access to the data on this line. The transformation from unknown -> list(list(list(int))) happens downstream.

Reply:
It does happen though, cfr 

https://github.com/wandb/weave/pull/140/files#diff-d81ba6a7c541ed19b444fabf09e9e74c7483b92b4b92dbd05fe2274b8a2c3e9cR157